lti_lib/student_score.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+
+function grading_grade(grade, payload) {
+
+    var headers = {
+        'Content-Type': 'application/json',
+    }
+    axios.post(payload["https://purl.imsglobal.org/spec/lti-ags/claim/endpoint"].lineitems, {
```

Comment:
"The scores service endpoint MUST be the **line item** resource url with the path appended with '/scores'." - you used lineitems (with an s) instead of lineitem (without an s) and you didn't append '/scores'